### DISCUSSION TITLE
Integrate with atlas for broader db support

### DISCUSSION THREAD

**Discussion by tomdavidson** [linkml/linkml#2064] [Ideas]
LinkML seems quite exciting but I am not using for a project due to only supporting SQLite, but perhaps integrating https://github.com/ariga/atlas by spitting out the the HCL or even wrapping (its APL2) would give LinkML a short cut to more relational db support along with the the migration functionality.

**Comment by cmungall** [comment]
Atlas looks like a great SQL migration tool!

We do support any sql alchemy dialect when generating tables:
https://linkml.io/linkml/generators/sqltable.html

However you are right that the SqlStore
https://linkml.io/linkml/developers/sqlstore.html
wrapper only binds to SQLite at the moment - in principle this should be
easy to extend.
